package.json

```diff
@@ -116,7 +116,7 @@
     "typescript": "4.6.3",
     "untildify": "4.0.0",
     "watch-detector": "1.0.1",
-    "web-ext": "6.8.0",
+    "web-ext": "^7.1.0",
```

Comment:
for future reference please pin the exact version

Reply:
Renovate does that but it's not merged yet: https://github.com/brave/web-discovery-project/pull/189